etc/fixtures/migrations/migrate-additional-sections.js

```diff
@@ -0,0 +1,55 @@
+const usage = `
+mongosh /tmp/fixtures/migrations/migrate-additional-sections.js
+    Script Options:
+        help: if true print this help message
+    
+    Run mongosh help for mongosh connection and authentication usage.
+    
+    Example: 
+    
+    mongosh --host localhost --port 27017 --eval "var help=true;" /tmp/fixtures/migrations/migrate-additional-sections.js
+    
+    docker exec -it <rosalution_db_container> mongosh /tmp/fixtures/migrations/migrate-additional-sections.js
+`
+
+if (help === true) {
+    print(usage);
+    quit(1);
+}
+
+if (typeof databaseName === 'undefined') {
+    databaseName = "rosalution_db";
+} else if (typeof databaseName !== 'string') {
+    print("databaseName must be a string");
+    quit(1);
+}
+
+db = db.getSiblingDB(databaseName);
+
+const annotationSectionsRename = {
+  'Modelability': 'Orthology',
+  'Protein Expression': 'Human Gene Expression',
```

Comment:
```suggestion
  'Protein Expression': 'Human_Gene_Expression',
```
The following target dataset name needs the underscores in it to match directly to what the configuration is looking for.